src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -343,17 +343,20 @@ private static String getCmdOutput(File workDir, String... cmd) throws Exception
     private static int runTestCommand(TestCommandArgs testArgs, JCommander jc) {
         // Figure out module name.
         // Join together spaced out names with underscores if necessary.
+        int returnCode = 131;
```

Comment:
131 represents "State not recoverable" in Linux. I guess defaulting to 1 "Operation not permitted" would be another option. Anything you see fit work for me.